bot/controllers/verify.js

```diff
@@ -0,0 +1,99 @@
+module.exports = () => {
+  const util = require('apex-util');
+
+  const _run = (message) => {
+    const ctrls = [{
+      cmd: '!verify',
+      example: '!verify <email_address>',
+      title: 'Verify Email Address',
+      desc: 'Verify user\'s email address',
+      showWithHelp: true,
+      posTargetUser: null,
+      posSecondaryCmd: null,
+      regexSplitCharacter: null,
+      allowInDM: true,
+      resType: 'dm',
+      action: (message, ctrl, msg) => {
+        const email = msg.parsed[1];
+        // TODO: Client-side email validation (*@fullsail.edu || *@student.fullsail.edu)
+        // Method to generate random numeric verification code
+        // Modified to fit style guide from this SO answer:
+        // https://stackoverflow.com/a/39774334
+        const generateCode = (n) => {
+          const add = 1;
+          let max = 12 - add;
+          let min = 0;
+          if (n > max) {
+            return generateCode(max) + generateCode(n - max);
+          }
+          max = 10 ** (n + add);
```

Comment:
When I ran this on my machine I got an unexpected token error for the double `**` in `max = 10 ** (n + add)`. 
Would suggest switching it to `max = 10 * (n + add)`.

Reply:
The `**` operator is exponents, so `10 ** (n + add)` is 10<sup>(n + add)</sup>. `10 * (n + add)` isn't equivalent and I don't know how it would affect the randomness of the resultant string. I'm only using this operator because the linter rejected `Math.pow()`. 

Can you give more detail on the error you got? Was it from the linter or something else?

Ultimately though if we wind up using a different code generator method this becomes a non-issue.

Reply:
Sure thing. This is what I got. 
'0|max      | /Users/Matt_Owens/Desktop/DWA/Max-Bot/bot/controllers/verify.js:30
0|max      |           max = 10 ** (n + add);
0|max      |                             ^
0|max      | SyntaxError: Unexpected token *
0|max      |     at Object.exports.runInThisContext (vm.js:53:16)
0|max      |     at Module._compile (module.js:513:28)
0|max      |     at Object.Module._extensions..js (module.js:550:10)
0|max      |     at Module.load (module.js:458:32)
0|max      |     at tryModuleLoad (module.js:417:12)
0|max      |     at Function.Module._load (module.js:409:3)
0|max      |     at Module.require (module.js:468:17)
0|max      |     at require (internal/module.js:20:19)
0|max      |     at fs.readdirSync.forEach (/Users/Matt_Owens/Desktop/DWA/Max-Bot/bot/controllers/index.js:22:18)
0|max      |     at Array.forEach (native)'

Reply:
Sure thing. This is what I got. 
```javascript
0|max      | /Users/Matt_Owens/Desktop/DWA/Max-Bot/bot/controllers/verify.js:30
0|max      |           max = 10 ** (n + add);
0|max      |                             ^
0|max      | SyntaxError: Unexpected token *
0|max      |     at Object.exports.runInThisContext (vm.js:53:16)
0|max      |     at Module._compile (module.js:513:28)
0|max      |     at Object.Module._extensions..js (module.js:550:10)
0|max      |     at Module.load (module.js:458:32)
0|max      |     at tryModuleLoad (module.js:417:12)
0|max      |     at Function.Module._load (module.js:409:3)
0|max      |     at Module.require (module.js:468:17)
0|max      |     at require (internal/module.js:20:19)
0|max      |     at fs.readdirSync.forEach (/Users/Matt_Owens/Desktop/DWA/Max-Bot/bot/controllers/index.js:22:18)
0|max      |     at Array.forEach (native)
````

Reply:
@buchholz-sean your logic seems right!

ES5 `Math.pow(10, n + add)`
ES6 `10 ** (n + add)`

when I run both I have the same result however for some reason the ES6 version is erroring out!